msteams-platform/sbs-calling-and-meeting.yml

```diff
@@ -155,6 +155,7 @@ items:
     1. Select your **Resource group** from the dropdown list.
 
         To create a new resource group, select **Create new**, enter resource name, select **OK**, and select required location from **New resource group location** dropdown list.
+        For more information see [Create resource group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resource-groups-portal#create-resource-groups)
```

Comment:
Link to create resource group is not required here, Line 157 already covers this step and there is a link in the UI. Remove.

Reply:
The link is removed as suggested.